ci/docker/wasm32-wasi/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM rust:1-slim
```

Comment:
I think it's ok to use the same base as the other images if the installation of `cargo-wasi` is removed.

Reply:
Done by moving the cargo-wasi install to the github workflow

It was not really working anyway since the `docker run` imports cargo & rustc from the parent container to the child and I was setting up the inner container. Probably a side-effect of container caching that was making sure things work. It probably wouldn't have worked on a cold build.